recipes/opencv/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+    "2.4.13.7":
+      folder: "2.x"
```

Comment:
```suggestion
  "2.4.13.7":
    folder: "2.x"
```

again ;)

Reply:
YAML is totally flexible in terms of number of spaces, why do you prefer 2 instead of 4?

Reply:
I see, because conandata.yml follows 2 spaces.

Reply:
Yep, and also because it's enforced in .editorconfig